### DISCUSSION TITLE
2023-10-19

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#681] [daily_meeting]
### 오늘의 할 일
데모데이 요구사항 만족
엽토 오전 관리자 페이지
백엔드 배포

### 개인 일정
노아 5~6시

**Comment by jaehee329** [comment]
### 체크인
7점 - 그냥그래요

**Comment by aak2075** [comment]
### 체크인

10점 - 아 거의 다 풀었는데

**Comment by woo-jk** [comment]
### 체크인
8점 - 비가 추적추적

**Comment by yeopto** [comment]
### 체크인

6점 - 뭐 때문인지 모르겠는데 머리가 너무 아픕니다 안압상승느낌

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 이제 면접 준비 달립니다.

**Comment by woosung1223** [comment]
### 체크인
7점 - 넘후졸리네요

**Comment by nlom0218** [comment]
### 체크인

9점 - 어제 보단 몸 컨디션이 좋아요

### 체크아웃

10점 - 시간 빠르군요!
